website/landingpage/public/blog/overview.html

```diff
@@ -16,7 +16,7 @@
     />
     <link rel="stylesheet" href="../assets/css/app.css" />
 
-    <link rel="alternate" title="Luigi Blog" type="application/json" href="../blog/feeds/feed.json" />
+    <link rel="alternate" title="My Feed" type="application/json" href="../blog/feeds/feed.json" />
```

Comment:
I feel like it should be another way around. Instead of "My Feed" there should be "Luigi Blog"?

Reply:
It was changed in https://github.com/SAP/luigi/pull/1497/commits/4a5ae274be4cfb285cd93a193ed2dc736a8a8bc1